src/group_gui.cpp

```diff
@@ -136,16 +136,16 @@ class VehicleGroupWindow : public BaseVehicleListWindow {
 
 	Dimension column_size[VGC_END]; ///< Size of the columns in the group list.
 
-	void AddChildren(GUIGroupList *source, GroupID parent, int indent)
+	void AddChildren(GUIGroupList &source, GroupID parent, int indent)
 	{
-		for (const Group *g : *source) {
+		for (const Group *g : source) {
```

Comment:
`Group &` here then? Or is that pushing it?

Reply:
No, it's a list of pointers, so that won't work.